packages/@aws-cdk/aws-apigatewayv2-integrations/README.md

```diff
@@ -149,6 +150,52 @@ const httpEndpoint = new HttpApi(stack, 'HttpProxyPrivateApi', {
 });
 ```
 
+### Request Parameters
+
+Request parameter mapping allows API requests from clients to be modified before they reach backend integrations.
+To use parameter mapping, you specify API request parameters to modify, and specify how to modify the parameters. 
+
+Request parameter mapping is supported through a set of helper methods.
+The following example renames a request header from header1 to header2 by generating mappings
+`append:header.header2: $request.header.header1` and `remove:header.header1: ''`:
+
+```ts
+const vpc = new ec2.Vpc(stack, 'VPC');
+const lb = new elbv2.ALB(stack, 'lb', { vpc });
+const listener = lb.addListener('listener', { port: 80 });
+listener.addTargets('target', {
+  port: 80,
+});
+
+const httpEndpoint = new HttpApi(stack, 'HttpProxyPrivateApi', {
+  defaultIntegration: new HttpAlbIntegration({
+    listener,
+    requestParameters: new ParameterMapping()
+      .appendHeader('header2', MappingValue.header('header1'))
+      .removeHeader('header1');
+    }),
+  }),
+});
+```
+
+
+
+To use values which does not yet have any helper functions, you can use the custom methods:
+
+```ts
+const httpEndpoint = new HttpApi(stack, 'HttpProxyPrivateApi', {
+  defaultIntegration: new HttpAlbIntegration({
+    listener,
+    requestParameters: new RequestParameters()
+      .addParameter({
+        mappingKey: MappingKey.custom('new.header'),
+        mappingValue: MappingValue.custom('new.value'),
+    }),
```

Comment:
Why not just

`new ParameterMapping().custom('new.header', MappingValue.custom('new.value'))?`

Reply:
Fixed

Reply:
But it's not fixed though? It's still the same in the README.